docs/usage/configuration-options.md

```diff
@@ -2765,11 +2765,11 @@ This is why we configured an upper limit for how long we wait until creating a P
 ## prPriority
 
 Sometimes Renovate needs to rate limit its creation of PRs, e.g. hourly or concurrent PR limits.
-In such cases it sorts/prioritizes by default based on the update type (e.g. patches raised before minor, minor before major).
+In such cases it sorts/prioritizes by default based on the update type (e.g. `patch` raised before `minor`, `minor` before `major`).
```

Comment:
This line feels clunky.

If I'm reading this right, if Renovate is rate-limited it prioritizes updates in this order: 

1. `patch`
2. `minor`
3. `major`

I expected it to be like this, before I read the docs:

1. `major`
2. `minor`
3. `patch`

I guess the philosophy here is: "Give them the easy updates first and worry about the big stuff later, when I'm no longer rate-limted."?

Reply:
Yes, easiest stuff first. Also it prioritizes regardless of whether it's rate limited or not

Reply:
```suggestion
By default, Renovate sorts/prioritizes based on the update type, going from smallest update to biggest update in order.
So Renovate will create `patch` updates first, then `minor` updates, and finally `major `updates.
```

How about this then?

Reply:
Yes, but actually it goes pinDigest, pin, digest, patch, minor, major

Reply:
```suggestion
By default, Renovate sorts/prioritizes based on the update type, going from smallest update to biggest update.
Renovate creates update PRs in this order:

1. `pinDigest`
1. `pin`
1. `digest`
1. `patch`
1. `minor`
1. `major`

```